io.github.yamanqd.antware.yml

```diff
@@ -0,0 +1,82 @@
+app-id: io.github.yamanqd.antware
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: run_antware.sh
+appdata-license: gpl-3.0
```

Comment:
```suggestion
```

Reply:
that's a valid tag, but I'm pretty sure the value must be a valid spdx identifier.

Reply:
I'd add the project_license here instead of in the manifest https://github.com/YamanQD/AntWare/blob/0071d7f7a06770764d4f453d25b131c14c5a6b23/io.github.yamanqd.antware.metainfo.xml#L9C42-L9C43 The appdata is already there.

Reply:
Okay I'll add the license tag in the metainfo file